sr_hand_config/common/placeholder.yaml

```diff
@@ -0,0 +1 @@
+# Delete this file once other files start appearing in this folder
```

Comment:
Just an idea: maybe convert this file to `README.md` then it can act both as a placeholder and also provide useful information about why this folder is needed and what contents you expect here.